From: James Vega <jamessan@debian.org>
Date: Fri, 18 Aug 2006 09:06:20 -0400
Subject: Detect the rst filetype using the contents of the file

Closes: #382541
---
 runtime/autoload/dist/script.vim | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/runtime/autoload/dist/script.vim b/runtime/autoload/dist/script.vim
index 859126f..7534ef4 100644
--- a/runtime/autoload/dist/script.vim
+++ b/runtime/autoload/dist/script.vim
@@ -423,6 +423,14 @@ def DetectFromText(line1: string)
   elseif line1 =~ 'exec\s\+\S*scheme' || line2 =~ 'exec\s\+\S*scheme'
     setl ft=scheme
 
+    # rst files
+  elseif line1 =~ '^\.\.\s\|^\s*restindex\s*$'
+	|| line2 =~ '^\.\.\s\|^\s*restindex\s*$'
+	|| line3 =~ '^\.\.\s\|^\s*restindex\s*$'
+	|| line4 =~ '^\.\.\s\|^\s*restindex\s*$'
+	|| line5 =~ '^\.\.\s\|^\s*restindex\s*$'
+    set ft=rst
+
     # Git output
   elseif line1 =~ '^\(commit\|tree\|object\) \x\{40,\}\>\|^tag \S\+$'
     setl ft=git
