Index: vim-7.3.000+hg~ee53a39d5896/runtime/syntax/grub.vim
===================================================================
--- vim-7.3.000+hg~ee53a39d5896.orig/runtime/syntax/grub.vim	2010-10-17 12:51:13.000000000 +0200
+++ vim-7.3.000+hg~ee53a39d5896/runtime/syntax/grub.vim	2010-10-17 12:51:29.000000000 +0200
@@ -40,7 +40,7 @@
                               \ displaymem embed find fstest geometry halt help
                               \ impsprobe initrd install ioprobe kernel lock
                               \ makeactive map md5crypt module modulenounzip pause
-                              \ quit reboot read root rootnoverify savedefault setup
+                              \ quiet quit reboot read root rootnoverify savedefault setup
                               \ testload testvbe uppermem vbeprobe
 
 syn keyword grubSpecial       saved
